File: 2016-09-06/beautiful_binary.py
#!/bin/python3

# HackerRank Problem: https://www.hackerrank.com/challenges/beautiful-binary-string

import sys


n = int(input().strip())
B = input().strip()

total = 0
while '010' in B:
    total += 1
    i = B.index('010')
    B = B[:i] + '111' + B[i+3:]
    
print(total )

print([i for i in range(10)])

File: 2016-09-06/beautiful_binary.java
// HackerRank Problem: https://www.hackerrank.com/challenges/beautiful-binary-string

import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner scan = new Scanner(System.in);
        scan.next();
        
        while (scan.hasNext()) {
            String scanned = scan.next();
            int i = 0;
            while (scanned.indexOf("010") > -1) {
                i++;
                scanned = scanned.replaceFirst("010","111");
            }
            System.out.println(i);
        }
    }
}